diff --git a/lib/linter/compiled-pane.js b/lib/linter/compiled-pane.js
index 8f93f74..f3588e3 100644
--- a/lib/linter/compiled-pane.js
+++ b/lib/linter/compiled-pane.js
@@ -7,6 +7,8 @@ import { CompositeDisposable } from 'atom'
 import PaneItem from '../util/pane-item'
 import { open } from '../util/opener'
 
+var subs
+
 // pane for side-to-side view of compiled code and source code
 export default class CompiledPane extends PaneItem {
   name = 'CompiledPane'
diff --git a/lib/linter/pane.js b/lib/linter/pane.js
index ba4239b..8d825c3 100644
--- a/lib/linter/pane.js
+++ b/lib/linter/pane.js
@@ -7,6 +7,8 @@ import { CompositeDisposable } from 'atom'
 import PaneItem from '../util/pane-item'
 import { open } from '../util/opener'
 
+var subs
+
 export default class LinterPane extends PaneItem {
   name = 'LinterPane'
 
